packages/factom-did-provider/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@sphereon/veramo-factom-did-provider",
+  "name": "@sphereon/sphereon-sdk-factom-did-provider",
```

Comment:
should this not be ssi-sdk-factom-did-provider?

Reply:
Yupz, good catch. I did notice it at one point myself, but forgot about it 😂